main.c

```diff
@@ -1,6 +1,6 @@
 #include "src/dunst.h"
 
-int main(int argc, char *argv[])
+int main(int argc, const char *argv[])
```

Comment:
Why only make argv const? I reckon argc also could be const

Reply:
It was due to const correctness somewhere along the chain. Also a int paramater doesn't really matter, but a pointer can be modified otherwise (and argv should not be modified)